models/order-alternative/src/generators.ts

```diff
@@ -0,0 +1,117 @@
+import { Company } from '@commercetools-test-data/business-unit';
+import { Cart, LineItem } from '@commercetools-test-data/cart';
+import { CartDiscount } from '@commercetools-test-data/cart-discount';
+import {
+  Address,
+  CentPrecisionMoney,
+  ClientLogging,
+  Reference,
+} from '@commercetools-test-data/commons';
+import {
+  fake,
+  Generator,
+  oneOf,
+  sequence,
+} from '@commercetools-test-data/core';
+import { CustomerGroup } from '@commercetools-test-data/customer-group';
+import { Quote } from '@commercetools-test-data/quote';
+import { State } from '@commercetools-test-data/state';
+import { Store } from '@commercetools-test-data/store';
+import { createRelatedDates } from '@commercetools-test-data/utils';
+import {
+  inventoryMode,
+  orderState,
+  paymentState,
+  shipmentState,
+  shippingMode,
+  taxCalculationMode,
+  taxMode,
+  taxRoundingMode,
+} from './constants';
+import { TOrderGraphql, TOrderRest } from './types';
+
+const [getOlderDate, getNewerDate] = createRelatedDates();
+
+const commonFieldsInitializers = {
+  id: fake((f) => f.string.uuid()),
+  key: fake((f) => f.lorem.words()),
+  version: sequence(),
+  createdAt: fake(getOlderDate),
+  lastModifiedAt: fake(getNewerDate),
+  createdBy: fake(() => ClientLogging.random()),
+  lastModifiedBy: fake(() => ClientLogging.random()),
+  // completedAt: null,
+  orderNumber: fake((f) => String(f.number.int({ min: 100000 }))),
+  customerId: fake((f) => f.string.uuid()),
+  customerEmail: fake((f) => f.internet.email()),
+  anonymousId: fake((f) => f.string.uuid()),
+  lineItems: fake(() => [LineItem.random()]),
+  customLineItems: [],
+  totalPrice: fake(() => CentPrecisionMoney.random()),
+  // taxedPrice: null,
+  // taxedShippingPrice: null,
+  shippingAddress: fake(() => Address.random()),
+  billingAddress: fake(() => Address.random()),
+  shippingMode: oneOf(...Object.values(shippingMode)),
+  shipping: [],
+  taxMode: oneOf(...Object.values(taxMode)),
+  taxRoundingMode: oneOf(...Object.values(taxRoundingMode)),
+  taxCalculationMode: oneOf(...Object.values(taxCalculationMode)),
+  country: fake((f) => f.location.countryCode()),
+  orderState: oneOf(...Object.values(orderState)),
+  shipmentState: oneOf(...Object.values(shipmentState)),
+  paymentState: oneOf(...Object.values(paymentState)),
+  // shippingInfo: null,
+  syncInfo: null,
```

Comment:
I think this was so that they're null in the GraphQL version. Just a guess.

Reply:
But that's not required, isn't it?

Reply:
Nope. Unless you request the field and it should return an "empty value".